web/concrete/src/File/Service/File.php

```diff
@@ -444,4 +444,30 @@ public function replaceExtension($filename, $extension)
 
         return $newFilename;
     }
+
+    /**
+     * Checks if two path are the same, considering directory separator and OS case sensitivity.
+     *
+     * @param string $path1
+     * @param string $path2
+     *
+     * @return bool
+     */
+    public function isSamePath($path1, $path2)
+    {
+        $path1 = str_replace(DIRECTORY_SEPARATOR, '/', $path1);
+        $path2 = str_replace(DIRECTORY_SEPARATOR, '/', $path2);
+        // Check if OS is case insensitive
```

Comment:
Maybe there's a better way to do this? Some magic constant like `PHP_EOL` or `DIRECTORY_SEPARATOR` that tells us of the OS is case insensitive?